packages/form-builder/addon/components/cfb-form-editor/general.hbs

```diff
@@ -74,7 +51,7 @@
 
     <div class="uk-text-right">
       <f.submit
-        @disabled={{or f.loading f.model.isInvalid}}
+        @disabled={{f.loading}}
```

Comment:
Intended change of behaviour?

Reply:
This was used for testing purposes, reverted it.

Reply:
Yes